tests/test.rfb.js

```diff
@@ -3265,6 +3265,11 @@ describe('Remote Frame Buffer protocol client', function () {
                     expect(spy).to.have.been.calledOnce;
                     expect(spy.args[0][0].detail.name).to.equal('som€ nam€');
                 });
+
+                it('should handle the extendedMouseButtons pseudo-encoding', function () {
+                    sendFbuMsg([{ x: 0, y: 0, width: 0, height: 0, encoding: -316 }], [[]], client);
+                    expect(client._extendedPointerEventSupported).to.equals(true);
```

Comment:
We do not want to look at internal stuff. We want to see changes in externally observed behaviour. I.e., what pointer events it will send.